setup_chroot.sh

```diff
@@ -88,7 +88,7 @@ build_chroot()
 
 	# Create our chroot
 	echo -e "\n${COLOR_ON}Bootstrap the chroot...${COLOR_OFF}" 
-	sudo -E debootstrap --arch=${pkg} --include=wget precise ${CHROOT_DIR}/${CHROOT_NAME} http://archive.ubuntu.com/ubuntu/
+	sudo -E debootstrap --arch=${pkg} --include=wget --keyring=/etc/apt/trusted.gpg precise ${CHROOT_DIR}/${CHROOT_NAME} http://archive.ubuntu.com/ubuntu/
```

Comment:
This still relies on `apt-key adv --recv-keys` to have downloaded the right keys, but that command does not perform any validation, so an attacker who can interfere with the network path could potentially substitute their own malicious keyring.

I think it's a better approach to include a copy of the known-good public key in this repo so we have a trust path to it (#100).